pages/_app.js

```diff
@@ -1,3 +1,5 @@
+import GlobalStyle from '../components/GlobalStyles';
+
 function MyApp({ Component, pageProps }) {
   return <Component {...pageProps} />;
 }
```

Comment:
You could already implement your GlobalStyles and you might want to consider renaming the component from `GlobalStyle` to `GlobalStyles`. It would then have the same name as the corresponding file :) 

```suggestion
import GlobalStyle from '../components/GlobalStyles';

function MyApp({ Component, pageProps }) {
  return (
  <>
  <GlobalStyle />
  <Component {...pageProps} />
  </>);
}
```

Reply:
thanks!